notebooks/cosine_io.ipynb

```diff
@@ -0,0 +1,1789 @@
+{
```

Comment:
<p>I would suggest to skip this variable entirely and just always directly reference <code>pr.storage_interface</code>.It only occurs three more times, and I think the reduced cognitive load of one less variable, and the clearer connection to the project will be worthwhile.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyiron/pyiron_contrib/pull/632/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>